tensorflow/python/saved_model/simple_load.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""SavedModel simple load functionality."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.saved_model import signature_constants
+from tensorflow.python.saved_model import tag_constants
+from tensorflow.python.util.tf_export import tf_export
+from tensorflow.python.saved_model import loader
+
+
+@tf_export('saved_model.simple_load')
+def simple_load(session, export_dir):
+  """Load a SavedModel artifact into a python TensorFlow Session.
+
+  This function assumes that the export directory contains a model that has been
+  serialized using the @{tf.saved_model.simple_save} function.
+
+  More information about SavedModel and signatures can be found here in the
+  @{$guide/saved_model} documentation.
```

Comment:
@MarkDaoust -- what's the new format for these magic doc decorators?

Reply:
Code references can just be back-quoted `tf.saved_model.simple_save` and they'll automatically be linked when the site is generates.

We haven't sorted out doc references yet, so that @{$guide/saved_model} should be good for now.